Add FAQ Section (#322)
helium/docs: docs/faq/security.mdx

---
id: security 
hide_title: true
title: Security FAQ
sidebar_label: Security
slug: /faq/security
---

# FAQ - Security

> The security of the network and Hotspot owners is of the utmost importance to us.  While the internet connection is necessary, it is only used to provide backhaul to the Hotspot for blockchain peer-to-peer and LongFi networking. As security concerns are ever present, here is what we have done, and are doing, to combat potential risks:

Reviewer: ```suggestion
The security of the network and Hotspot owners is of the utmost importance to us.  While the Internet connection is necessary, it is only used to provide backhaul to the Hotspot for blockchain peer-to-peer and LongFi networking. As security concerns are ever present, here is what we have done, and are doing, to combat potential risks:
```

> - The most common attack vector to get into a device like the Hotspot is inbound ports.  The Helium Hotspot only requires one port open in both directions (TCP Port 44158), so all other inbound ports can be secured behind a firewall per your personal security needs.  

Reviewer: ```suggestion
- The most common attack vector to get into a device like the Hotspot is inbound ports.  The Helium Hotspot only requires one port open in both directions - **TCP Port 44158**. All other inbound ports can be secured behind a firewall per your personal security needs.  
```

> - Helium devices are hardware secured to protect the traffic from the utilized spectrum.  This means the security is built-in since devices using the network have AES private key encryption at the chip level.  

Reviewer: ```suggestion
- Helium-enabled LoRaWAN devices are hardware-secured to protect the traffic from the utilized spectrum.  This means the security is built-in since devices using the network have AES private key encryption at the chip level.  
```

> - Helium hardware and network comply with all FCC regulations for operating in the unlicensed 902-928MHz spectrum in the US, and has the CE mark for the European 868MHz spectrum.

Reviewer: ```suggestion
- All approved Helium Hotspots comply with the relevant radio and wireless regulations in the country or region where they are deployed.  For example, Hotspots in the US comply with FCC regulations for operating in the unlicensed 902-928MHz spectrum, the CE mark for the European 868MHz spectrum, etc.
```

> - Expected bandwidth utilized by devices is a meager 1-5kb/s, about the size of a long text message, and devices communicate with the Hotspot over LongFi, not IP (Internet Protocol).

Reviewer: ```suggestion
- Expected bandwidth utilized by devices is 1-5kb/s (about the size of a long text message) and devices communicate with the Hotspot over LongFi, not IP (Internet Protocol).
```

> - While very reasonable in price, using the Helium network is not free. This is inherently a deterrent for potential scammers.

Reviewer: ```suggestion
- While very inexpensive to use, the Helium network is not free. Any transaction requires data credits. This is inherently a deterrent for potential scammers.
```
